docs/mddocs/Quickstart/ollama_portablze_zip_quickstart.md

```diff
@@ -5,39 +5,44 @@
 
 This guide demonstrates how to use [Ollama portable zip](https://github.com/intel/ipex-llm/releases/tag/v2.2.0-nightly) to directly run Ollama on Intel GPU with `ipex-llm` (without the need of manual installations).
 
-> [!NOTE]
-> Currently, IPEX-LLM only provides Ollama portable zip on Windows.
-
 ## Table of Contents
-- [Prerequisites](#prerequisitesa)
+### Windows Quickstart
+- [Prerequisites](#prerequisites)
 - [Step 1: Download and Unzip](#step-1-download-and-unzip)
 - [Step 2: Start Ollama Serve](#step-2-start-ollama-serve)
 - [Step 3: Run Ollama](#step-3-run-ollama)
-- [Tips & Troubleshooting](#tips--troubleshootings)
+- [Tips & Troubleshooting](#tips--troubleshooting)
```

Comment:
`Tips & Troubleshooting` should be common across Windows & Linux - need to also add model source support to Linux